contrail-provisioner/main.go

```diff
@@ -829,8 +829,15 @@ func databaseNodes(contrailClient *contrail.Client, nodeList []*types.DatabaseNo
 }
 
 func setupAuthKeystone(client *contrail.Client, keystoneAuthParameters *KeystoneAuthParameters) {
+	// CEM-17791 - AddEncryption expects http url
```

Comment:
This a public code so I'm not sure if we should expose our internal Jira tickets

Reply:
Pointed to the commit which is already merged, not sure what version of the contrail-go-api we use, but if we could use the newest, then this addition wouldn't be needed.